### PR TITLE
Add compliant and noncompliant examples of python/pytorch-disable-gradient-calculation@v1.0

### PR DESCRIPTION
**Pull request by swaraj-amazon**
*Description of changes:*
* Rules Added: python/pytorch-disable-gradient-calculation@v1.0
* Added compliant and non compliant examples

### CODE REVIEW

**Review by san2488 — APPROVED** [review]

File: src/python/detectors/pytorch-disable-gradient-calculation/pytorch-disable-gradient-calculation.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=pytorch-disable-gradient-calculation@v1.0 defects=1}
+def disable_gradient_calculation_noncompliant():
+    import torch
+    # Noncompliant: disables gradient calculation using `torch.no_grad()`.
+    with torch.no_grad():
+        model.eval()
```
Comment (linghuiluo) [inline src/python/detectors/pytorch-disable-gradient-calculation/pytorch-disable-gradient-calculation.py:9]: add # some code under model.eval()
